examples/wcf/client-core/Examples.Wcf.Client.Core.csproj

```diff
@@ -1,16 +1,16 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Based on file name, here you should have net6.0/net7.0. .NET Framework is handled by examples/wcf/client-netframework/Examples.Wcf.Client.NetFramework.csproj

Reply:
Good catch! Now that `netcoreapp3.1` is not supported, I think we could rename this project to `Examples.Wcf.Client.DotNet` and have `net6.0` as the TargetFramework. What do you think?

Reply:
I have updated it.